src/main/index.ts

```diff
@@ -1,16 +1,24 @@
 import { app, BrowserWindow } from 'electron';
+import { configStore } from '../shared/configureStore';
 declare const MAIN_WINDOW_WEBPACK_ENTRY: any;
 
 // Handle creating/removing shortcuts on Windows when installing/uninstalling.
 if (require('electron-squirrel-startup')) {
   app.quit();
 }
 
+const store = configStore('main');
+
 const createWindow = (): void => {
+  store.subscribe(() => console.log('action received in main'));
```

Comment:
This should be deleted, logs the text in our terminal (not browser console, as it is the main process)